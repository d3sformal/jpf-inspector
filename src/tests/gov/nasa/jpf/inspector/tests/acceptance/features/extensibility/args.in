arg1 20
less_than 20 50 80
garbage_breakpoint 20 < 40
garbage_breakpoint 20 < 40 and instruction_type = any
show breakpoint