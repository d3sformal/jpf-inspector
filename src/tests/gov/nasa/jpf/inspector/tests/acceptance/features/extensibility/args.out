New assertion successfully created with ID 2.
1 :  state=ENABLED hits=0 hitsTotal=0 position=*:20
2 :  state=ENABLED hits=0 hitsTotal=0 assert ( *:20 ) ( (50 < 80) )
3 :  state=ENABLED hits=0 hitsTotal=0 ( garbage_collection=begin ) and ( (20 < 40) )
4 :  state=ENABLED hits=0 hitsTotal=0 ( garbage_collection=begin ) and ( ( (20 < 40) ) and ( instruction_type=any ) )